tests/server.filters.unit.test.js

```diff
@@ -0,0 +1,117 @@
+var common = require('./common.js'),
+    assert = require('assert'),
+    RadarTypes = require('../core/lib/type.js'),
+    controlMessage = { 
+      to: 'control:/dev/test',
+      op: 'nameSync',
+      options: {
+        association: {
+          id: 1,
+          name: 1
+        }
+      }
+    },
+    radarServer,
+    socket;
+
+describe('given a server with filters', function() {
+  beforeEach(function(done) {
+    radarServer = common.createRadarServer(done);
+    socket = { id: 1 };
+  });
+
+  afterEach(function(done) {
+    radarServer.terminate(done);
+  });
+
+  describe('with no filters', function() {
+    it('should not halt execution', function(done) {
+      socket.send = function(message) {
+        assert.equal('ack', message.op);
+        done();
+      };
+
+      radarServer._processMessage(socket, controlMessage);
+    });
+  });
+
+  describe('with 1 filter', function() {
+    it('if OK, it should run it and continue', function(done) {
```

Comment:
these tests are not really testing for halt/continue.. Just that the filter itself is called or not.